### ISSUE TITLE
Missing animation option

### ISSUE THREAD

**Issue by vinilneves** [WeakAuras/WeakAuras2#2231]
## Describe the bug
Expected 'custom function' to be an option at `Animations > Main > Type`, as described at the Wiki: https://github.com/WeakAuras/WeakAuras2/wiki/Custom-Animation-Functions

### To Reproduce
What I did:
1 - Create a bar aura for a buff using a template, fingers of frost in this case.
2 - Go to the animations tab 
3 - Click the `Type` dropdown under the `Main` section

**Screenshots**
![image](https://user-images.githubusercontent.com/9133888/84675005-0ceee680-af02-11ea-8326-54ab23414244.png)

![image](https://user-images.githubusercontent.com/9133888/84674586-85a17300-af01-11ea-9b90-887135dc380f.png)

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**
Happened with or without any other AddOns

**Which version of WeakAuras are you using?**
2.17.5
**Are you on World of Warcraft Classic or Retail?**

- [ ] Classic
- [x] Retail

**Was it working in a previous version? If yes, which was the last good one?**
Did not try

**Comment by asaka-wa** [comment]
The Custom Functions are defined in the individual animated properties, like Alpha or Colour.

**Comment by emptyrivers** [comment]
This is not a bug. In a custom animation, you can set multiple different aspects of the aura to be animated, which are the fade, color, scale, etc options. Those are where custom code is accepted.

**Comment by vinilneves** [comment]
Nope, it doesnt accept custom functions, as stated on the Wiki...

**Comment by vinilneves** [comment]
At least the wiki should be changed, if that's all it is available

**Comment by vinilneves** [comment]
Think I got it...Should I post my example here for reference? How about adding examples like this to the wiki?

**Comment by vinilneves** [comment]
What I did to solve this:

1. Add two `Conditions`, one for `Remaining Duration` <= my threshold and the other one if >
2. Set them to `Run Custom Code` 
3. Set a custom variable to `true` or `false` according to the condition, I used `aura_env.brain_freeeze_ending`
![image](https://user-images.githubusercontent.com/9133888/84681640-cce03180-af0a-11ea-830b-96bb8f6ef619.png)
4. Go to `Animations` and select `Custom` for the `Type` under `Main`
5. Check for example `Fade` and change it's `Type` to `Custom Function`
6. Change the function so it returns values based on the custom variable set by your `Actions`
![image](https://user-images.githubusercontent.com/9133888/84683653-eb93f780-af0d-11ea-9e45-c3db798186ef.png)

```lua
function(progress, start, delta)
    if aura_env.brain_freeeze_ending then
        return start + (progress * delta)
    end
    return start
end
```
